run_snafu.py

```diff
@@ -48,30 +48,30 @@ def main():
     index_args.prefix = "snafu-%s" % index_args.tool
     # set up a standard format for time
     FMT = '%Y-%m-%dT%H:%M:%SGMT'
-    
-    #instantiate elasticsearch instance and check connection 
+
+    #instantiate elasticsearch instance and check connection
     es={}
     if "es" in os.environ:
         es['server'] = os.environ["es"]
         es['port'] = os.environ["es_port"]
         index_args.prefix = os.environ["es_index"]
         index_args.index_results = True
-    
-        es = elasticsearch.Elasticsearch([
-        {'host': es['server'],'port': es['port'] }],send_get_body_as='POST')
-    
-        if not es.ping():
-            logger.warn("Elasticsearch connection failed or passed incorrectly, turning off indexing")
+        try:
+            _es_connection_string = str(host) + ':' + str(port)
+            es = elasticsearch.Elasticsearch([_es_connection_string],send_get_body_as='POST')
+            logger.info("Connected to the elasticsearch cluster with info as follows:" + str(es.info()))
+        except Exception as e:
```

Comment:
I don't think the exception will be thrown here.  It will be thrown when you try to post results to an index.   I commented out the es.ping() call in run_snafu and that's what happened.   That's why Alex was calling es.ping(), to try to find out if there really was a connection or not, right?

Reply:
the exception will be thrown here when es.info() doesn't work, and in our case i gave the user snafu monitoring permissions too, so it'll help us catch the issue if something changes. making es.ping() work requires additional permissions to the snafu user, which would complicate things, vs for es.info() which only requires monitoring permission.